.github/workflows/chromatic.yml

```
@@ -7,7 +7,8 @@ on:
     branches:
       - main
     paths:
-      - "stories/**"
+      - 'stories/**'
+      - 'packages/**'
```

Should this be:

```suggestion
      - 'packages/*/src/**/*.stories.*'
```

for specificity?

---

🤔 or should we just enable it as is for really any `src` changes in case that impacts Storybook?

---

This pattern is actually used for marking _when_ to trigger Chromatic, which we want to do any time we change `packages/**`.

We do have a glob pattern in `.storybook/main.js` which finds all stories to include. 
I'll update it to EXCLUDE all the `v37` stories that shouldn't be included yet.